compliance/google/unlabeled_resources/unlabeled_resources.pt

```diff
@@ -0,0 +1,425 @@
+          var isResMissedTag=false;
+          var all_include_label='';
+          var all_missed_label='';			  
+          if(JSON.stringify(obj.items[key]["labels"]) == undefined){
+             var labels="none";
+           }else{
+             var labels=obj.items[key]["labels"];	
+             var labels_str=JSON.stringify(labels);			  
+           }
+          for(var j=0; j < param_label_key_lower.length; j++){		  
+            if(_.isEmpty(labels_str) || (labels_str.toLowerCase().indexOf(param_label_key_lower[j]))==-1){
+              isResMissedTag=true;
+              all_missed_label=all_missed_label+" "+param_label_key_lower[j];
+            }else{
+              all_include_label=all_include_label+" "+param_label_key_lower[j];
+            }
+          }
+          if(isResMissedTag){				
+           result.push({
+            id: obj.items[key]["id"], 
+            labels: labels,
+            labelFingerprint: obj.items[key]["labelFingerprint"],
+            selfLink: obj.items[key]["selfLink"],
+            all_missed_label: all_missed_label,
+            all_include_label: all_include_label,
+            labels_str: labels_str,
+            type: type,
+            name: obj.items[key]["name"]			 
+           });
+          }
+       }
+     }
+   }				  
+ })	
+
+EOS
+end
+
+script "js_resource_uri", type: "javascript" do
+ parameters "ds_google_project"
+  result "result"
+  code <<-EOF
+    var result=[];
+    _.each(ds_google_project, function(project){	
+
+   var resource_uri = {
+	                    uri: project.projectId+"/aggregated/disks"
+                       };
+    result.push(resource_uri);	
+    resource_uri = {                        						
+                    uri: project.projectId+"/global/externalVpnGateways"
+                    }
+    result.push(resource_uri);					
+    resource_uri = {						
+                    uri: project.projectId+"/global/images"
+                   };
+    result.push(resource_uri);
+    resource_uri = {						
+                    uri: project.projectId+"/aggregated/instances"
+                   }; 
+    result.push(resource_uri);				   
+    resource_uri = {						
+                    uri: project.projectId+"/global/snapshots"
+                   };
+    result.push(resource_uri);				   
+    resource_uri = {						   
+                    uri: project.projectId+"/aggregated/vpnGateways"
+                   };
+    result.push(resource_uri);
+	});					
+EOF
+end
+
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_azure_resource" do
+  validate $ds_filtered_resources do
+    summary_template "{{ len data }} Google Resources out of compliance."
+    escalate $esc_email
+    escalate $esc_label_resources
+    check eq(size(data),0) 
+    export do
+      resource_level true
+      field "id" do
+        label "Resources Id"
```

Comment:
Lets move resource id to the last column.  Make Resource Type the first column, then Resource name.